layout/_spacing.scss

```diff
@@ -8,39 +8,51 @@
 /* 
 	Margin Helpers
 */
-.mg { margin-top: 0.5em; margin-bottom: 0.5em; }
-.mg--big { margin-top: 1.5em; margin-bottom: 1.5em;}
-.mg--double { margin-top: 1em; margin-bottom: 1em; }
-.mg--small { margin-top: 0.25em; margin-bottom: 0.25em; }
+.mg { margin-top: $base-unit; margin-bottom: $base-unit; }
```

Comment:
All of these units are now larger, based on the 'base unit' values.  I think it makes more sense this way.